src/BatteryStateOfChargeService.cpp

```diff
@@ -26,6 +26,10 @@
  */
 
 #include "BatteryStateOfChargeService.h"
+#include "BatteryData.h"
+#include <QTextStream>
+#include <QDebug>
+#include <QFile>
```

Comment:
Lines 30-32 are not being used in this file, so they should be removed to not confuse a reader as to their purpose.